src/types/CharData.d.ts

```diff
@@ -0,0 +1,62 @@
+import firebase from "firebase";
+import DocumentReference = firebase.firestore.DocumentReference;
+
+export default interface CharData {
+  id?: string; // キャラクターの一意のID
+  user?: string; // キャラクター作成者
+  createDate?: number; // 作成日 経過ミリ秒の文字列でいいのかな？
```

Comment:
```suggestion
  createDate?: Date;
```

日付は`Date`にしたい。firestoreにもそのまま保存できる。

- https://firebase.google.com/docs/firestore/manage-data/add-data?hl=ja
- https://developer.mozilla.org/ja/docs/Web/JavaScript/Reference/Global_Objects/Date

Reply:
修正しました